### ISSUE TITLE
Improve the display options when multiple sync configurations are found

### ISSUE THREAD

**Issue by matthew-muscat** [devspace-cloud/devspace#1300]
**Is your feature request related to a problem?**  
When attempting a sync using a `devspace.yaml` with multiple sync configurations, the CLI presents a prompt to select one of the sync configurations to be utilised.

The display of the available options can be a little vague in information, in that it does not provide uniquely identifable information as to which container / image you are making a selection of.

```
> devspace sync --config=devspace.yaml
[info]   Using namespace 'example'
[info]   Using kube context 'gke_internal-au-se1'

? Multiple sync configurations found. Which one do you want to use?
  [Use arrows to move, space to select, type to filter]
> 0: Sync ./ (local) <-> /srv (container)
  1: Sync ./ (local) <-> /srv (container)
  2: Sync ./public (local) <-> /export/public (container)
```

**Which solution do you suggest?**  
The messaging in the options should include an unique identifier for the sync entry — this identifier could be the sync configuration's `imageName`, `labelSelector` or `containerName` configuration (any / all of which are defined for each sync configuration)

**Which alternative solutions exist?**  
Using the sync command, it's possible to declare on the command line using `--container` or `--label-selector` — but that's moreso a workaround rather than a solution to the display of the selection options

/kind feature

**Comment by FabianKramm** [comment]
@matthew-muscat thanks for reporting this issue! Yes that makes sense, we can add the image name and label selector to the option text.

**Comment by matthew-muscat** [comment]
Thanks @FabianKramm for getting this change introduced — much appreciated.

As these changes were merged in as part of #1302 and released with [v5.7.3](https://github.com/devspace-cloud/devspace/releases/tag/v5.7.3) — I'll be closing this request
